code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -130,6 +130,31 @@
 				L.apply_damage(trap_damage, BRUTE)
 	..()
 
+/obj/item/restraints/legcuffs/beartrap/on_found(mob/finder)
+	if(!armed)
+		return FALSE
+	var/mob/living/L = finder
+	armed = FALSE
+	update_icon()
+	playsound(src.loc, 'sound/effects/snap.ogg', 50, 1)
+	if(!silent_arming)
+		L.visible_message("<span class='danger'>[L] triggers \the [src].</span>", "<span class='userdanger'>You trigger \the [src]!</span>")
```

Comment:
```suggestion
		L.visible_message("<span class='danger'>[L] triggers [src].</span>", "<span class='userdanger'>You trigger [src]!</span>")
```
\the is implicit, no need to have these (Yeah I know it's not your code but still)